db/migrate/20150511140404_create_popup_notifications.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePopupNotifications < ActiveRecord::Migration
+  def change
+    create_table :popup_notifications do |t|
+      t.string :position
+      t.string :title
```

Comment:
I got a feeling maybe we should use views to show this. Please look at the `public_activity` gem to see how they handle notifications. Storing translatable strings in the database is a bad idea.